cmd/tuf-client/main.go

```diff
@@ -32,7 +33,7 @@ See "tuf-client help <command>" for more information on a specific command.
 
 	if cmd == "help" {
 		if len(cmdArgs) == 0 { // `tuf-client help`
-			fmt.Println(usage)
+			fmt.Fprint(os.Stderr, usage)
```

Comment:
why is usage going to stderr?

Reply:
🤷  I've seen it both ways. There are some tools that define an explicit, **strict** "everything that gets logged to stdout must be computer-readable" philosophy and apply that to help text too. Many tools do, of course, print usage on STDOUT.

See for instance [this request for usage on STDERR](https://github.com/cbeust/jcommander/issues/339). I particularly like the recommendation to follow [picocli](https://picocli.info/) (just one example):

> when the user specified invalid input, the error message and usage help should be printed to stderr, but when the user requests help with <cmd> --help, the usage help should be printed to stdout so users can pipe the output to tools like grep and less.

Reply:
Ah, thank you for the comment: Yeah, I think I originally assumed that help should be part of STDERR, but here, since the user explicitly requested for help, I think I concur, it should be STDOUT. I'll put out a fix PR.